### PR TITLE
Added Wallet Collection to firestore

### PR DESCRIPTION
**Pull request by ewelina-slepko**

### CODE REVIEW

File: src/app/authentication/authentication.service.ts [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+              private router: Router,
+              private notifierService: NotifierService) {
+  }
+
+  signUpWithEmailAndPassword(email, password) {
+    return this.auth.createUserWithEmailAndPassword(email, password).catch((error) => {
+      if (error) {
+        this.notifierService.notify(
+          error.message,
+          2,
```
Comment (FilipTalaga) [inline src/app/authentication/authentication.service.ts:23]: Use enum please

File: src/app/authentication/authentication.service.ts [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  currentUser: firebase.User;
+
+  constructor(public auth: AngularFireAuth,
+              private router: Router,
+              private notifierService: NotifierService) {
+  }
+
+  signUpWithEmailAndPassword(email, password) {
+    return this.auth.createUserWithEmailAndPassword(email, password).catch((error) => {
+      if (error) {
```
Comment (FilipTalaga) [inline src/app/authentication/authentication.service.ts:20]: I don't think that's necessary.

File: src/app/authentication/authentication.service.ts [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+  signInWithEmailAndPassword(email, password): Promise<firebase.auth.UserCredential | void> {
+    return this.auth.signInWithEmailAndPassword(email, password)
+      .catch((error) => {
+        console.log(error);
+        this.notifierService.notify('There is no user record corresponding to this identifier.', 2);
+      });
+  }
+
+  signOut() {
+    this.auth.signOut().then(res => console.log(res));
```
Comment (FilipTalaga) [inline src/app/authentication/authentication.service.ts:38]: Why the console log?

**Comment by ewelina-slepko** [comment]
Done!

**Pull request merged**